src/app/issuer/components/issuer-create/issuer-create.component.ts

```diff
@@ -38,12 +42,15 @@ export class IssuerCreateComponent extends BaseAuthenticatedRoutableComponent im
 		.addControl('issuer_street', '')
 		.addControl('issuer_streetnumber', '')
 		.addControl('issuer_zip', '')
-		.addControl('issuer_city', '');
+		.addControl('issuer_city', '')
+		.addControl('captcha', '');
```

Comment:
When skipping captcha box and pressing create account, no msg/attention raised to inform user to verify `not robot` first